app/about_us/OurCrew.js

```diff
@@ -0,0 +1,91 @@
+import styles from "./OurCrew.module.css";
+const OurCrew = () => {
+  return (
+    <>
+      <p className={styles["crew-p"]}>
+        Our crew is the heart and soul of Galactica. We are a diverse team of seasoned space
+        explorers, engineers, and visionaries who are united by a common goal: to make space travel
+        accessible and exciting for all.
+      </p>
+      <div className={styles["crew-cards"]}>
+        <div className={styles["crew-card"]}>
+          <img src="/crew/img-2.png" alt="Sarah Vega" className={styles["crew-card-img"]} />
```

Comment:
This `alt="..."` text is not providing any extra info for the sighted users, so you can either change it or remove it.
(We are already having the crew members name in the `<h2>` below.)
- More info on alt tag: https://www.w3.org/WAI/tutorials/images/
- Videos about Accessibility testing, and how people use assistive technologies:
  - Short intro video - https://www.youtube.com/watch?v=HE2R86EZPMA
  - Longer and detailed video from a non-sighted user - https://www.youtube.com/watch?v=OUDV1gqs9GA

Reply:
ah yeah make sense. Thanks :D